### PR TITLE
Add yum autoremove to make the image just a tiny bit smaller

### PR DESCRIPTION
**Pull request by xeraa**
No test added, but not sure it makes a lot of sense on this level.

### CODE REVIEW

File: templates/Dockerfile.j2 [hunk 1]
```diff
@@ -111,7 +111,8 @@ ENV JAVA_HOME /usr/lib/jvm/jre-1.8.0-openjdk
 
 RUN yum update -y && \
     yum install -y nc java-1.8.0-openjdk-headless unzip wget which && \
-    yum clean all
+    yum clean all && \
+    yum autoremove
```
Comment (dliappis) [inline templates/Dockerfile.j2]: This will require `-y` otherwise it blocks asking for confirmation.

**Comment by xeraa** [comment]
Right, thanks!

**Comment by dliappis** [comment]
@xeraa I am going to revert this PR as unfortunately `yum -y autoremove` ends up removing also useful packages for the test suite, such as `hostname`:

```
[root@2f7b4a2cc91a elasticsearch]# yum -y autoremove
Loaded plugins: fastestmirror, ovl
Resolving Dependencies
--> Running transaction check
---> Package bind-license.noarch 32:9.9.4-51.el7_4.1 will be erased
---> Package dbus-python.x86_64 0:1.1.1-9.el7 will be erased
---> Package hostname.x86_64 0:3.13-3.el7 will be erased
---> Package python-gobject-base.x86_64 0:3.22.0-1.el7_4.1 will be erased
--> Finished Dependency Resolution
--> Finding unneeded leftover dependencies
```

**Comment by xeraa** [comment]
Interesting. I thought that shouldn't happen. Guess we won't be able to use this "optimization" then

**Pull request merged**